src/modules/map/components/cesium/CesiumMap.vue

```diff
@@ -253,7 +247,9 @@ export default {
 
         this.viewerCreated = true
 
-        this.viewer.scene.postRender.addEventListener(limitCameraCenter(TILEGRID_EXTENT_EPSG_4326))
+        this.viewer.scene.postRender.addEventListener(
+            limitCameraCenter(LV95.getBoundsAs(WGS84).flatten)
+        )
```

Comment:
Similar comment as down below, would it make sense to have config value for the default bound/center ? To avoid hardcoding it here, so our default bounds and center is CH but for other it might something else. So could we have something like this

```javascript
// CesiumMap.vue
limitCameraCenter(DEFAULT_PROJECTION.getBoundsAs(WGS84).flatten)
```

Maybe then also add a comment in the config.js that the default projection is also used as default bound and center.

Reply:
maybe we should do that only when the `DEFAULT_PROJECTION` is a `CustomCoordinateSystem`, as it would normally be the case that such a system will shrink its size to fit some part of the World.